src/bin/pg_autoctl/cli_service.c

```diff
@@ -30,6 +30,7 @@ static int stop_signal = SIGTERM;
 static void cli_service_run(int argc, char **argv);
 static void cli_keeper_run(int argc, char **argv);
 static void cli_monitor_run(int argc, char **argv);
+static void ensure_monitor_running(Monitor *monitor, MonitorConfig *mconfig);
```

Comment:
I think `ensure_monitor_pg_is_running` would be a better name (use of verb, specialise to the Postgres instance of the monitor). And then, why are we not using `ensure_local_postgres_is_running`?

Reply:
`ensure_local_postgres_is_running` works on a much different data structure and performs additional checks. I was not sure about merging two functions